bedrock/firefox/views.py

```diff
@@ -443,7 +443,13 @@ def hello(request):
     }
 
     if (waffle.switch_is_active('firefox-hello-2016')):
-        return l10n_utils.render(request, 'firefox/hello/index-2016.html')
+        variant = request.GET.get('v', '')
+        if variant in ['b']:
+            template = 'firefox/hello/index-2016-{0}.html'.format(variant)
```

Comment:
Not a blocker, but for future reference, could do this a bit simpler when there's only one available variant:

```
if variant == 'b':
    template = 'firefox/hello/index-2016-b.html'
```